### PR TITLE
Convert correctly mashes to yaml

### LINKED ISSUE #58: Exclamation marks in ACL policies generated from attributes
Chef 13.1.31 generate wrong ACL policies based on attributes:

```
#
# This file is generated by Chef
#
---
description: Admin, all access.
context: !ruby/hash:Mash
  project: ".*"
for: !ruby/hash:Mash
  resource:
  - !ruby/hash:Mash
    allow: "*"
  adhoc:
  - !ruby/hash:Mash
    allow: "*"
  job:
  - !ruby/hash:Mash
    allow: "*"
  node:
  - !ruby/hash:Mash
    allow: "*"
by: !ruby/hash:Mash
```

The hack in `aclpolicy.erb` that should fix these exclamation marks doesn't work now.

Rundeck 2.8.2

error:
```
Can't construct a java object for tag:yaml.org,2002:com.dtolabs.rundeck.core.authorization.providers.yaml.model.ACLPolicyDoc; exception=Cannot create property=context for JavaBean=Policy{context=null, description='Admin, all access.', for=null, by=null, id='null'}; null; Can't construct a java object for !ruby/hash:Mash; exception=Invalid tag: !ruby/hash:Mash
 in "<reader>", line 5, column 1:
    description: Admin, all access.
    ^
```

### PR DESCRIPTION
**Pull request by kamaradclimber**
Fixes #58

### CODE REVIEW

**Comment by kamaradclimber** [comment]
cc @aboten @criteo-cookbooks/sre-core

File: templates/default/aclpolicy.erb [hunk 1]
```diff
@@ -5,5 +5,30 @@
...
+      convert_to_hash(value)
+    end
+  when Array
+    el.map do |value|
+      convert_to_hash(value)
+    end
+  else
+    el
+  end
+end
```
Comment (jmauro) [inline templates/default/aclpolicy.erb]: It would be better to create library to achieve this part, don't you think?

Comment (kamaradclimber) [inline templates/default/aclpolicy.erb]: yes of course, feel free to do it :P

Comment (kamaradclimber) [inline templates/default/aclpolicy.erb]: done

**Comment by scopenco** [comment]
Can you release new ver to supermarket?

**Comment by kamaradclimber** [comment]
@scopenco will be done by https://travis-ci.org/criteo-cookbooks/rundeck-server/builds/243553268

**Pull request merged**